test-cypress/integration/help/rest-api.spec.js

```diff
@@ -0,0 +1,110 @@
+describe('Help / REST API', () => {
+
+    before(() => {
+        // Disables the security (only for this spec) to be able to interact with Swagger's iframe
```

Comment:
I observed that this will not always work but I'm not sure if it wasn't something from my end. 
Let's see how it behaves in Travis. If unstable, I'll add it to cypress.json